theme/open/views/help/people.hbs

```diff
@@ -19,7 +19,7 @@
 						</div>
 						<div id="collapse-0" class="collapse show in">
 							<div class="card-block">
-								Für die Meldung von Problemen oder Wünschen hinsichtlich der Bedienung der HPI Schul-Cloud verwende bitte das Kontaktformular im Hilfebereich oder sende eine E-Mail an <br><a href="mailto:feedback@hpi-schul-cloud.de" target="blank" rel="noopener" class="externalLink">feedback@hpi-schul-cloud.de</a>
+								Für die Meldung von Problemen oder Wünschen hinsichtlich der Bedienung der HPI Schul-Cloud verwende bitte das Kontaktformular im Hilfebereich oder sende eine E-Mail an <br><a href="mailto:dataportlernen.cloud@dataport.de" target="blank" rel="noopener" class="externalLink">dataportlernen.cloud@dataport.de</a>
```

Comment:
Better with translation

Reply:
Translation is not part of the ticket. In addition, it appears that the site is no longer live.